src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+
+    public BulldozerProducer() {
+    }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The method `get()` should return a `List<Bulldozer>` instead of a raw `List<Machine>` type to avoid using raw types and to ensure type safety.